components/timeline/TimelineItem.tsx

```diff
@@ -58,6 +60,7 @@ const TimelineItem: React.SFC<TimeLineItemProps> = props => (
 TimelineItem.defaultProps = {
   color: 'blue',
   pending: false,
+  position: 'left',
```

Comment:
should follow `<Timeline mode="xxx">`, don't set in defaultProps.

Reply:
After the mode is set, the position will be invalid.